api/src/Controller/Report/DecisionController.php

```diff
@@ -49,7 +46,7 @@ public function upsertDecision(Request $request)
             $this->denyAccessIfReportDoesNotBelongToUser($report);
             $decision = new EntityDir\Report\Decision();
             $decision->setReport($report);
-            $report->setReasonForNoDecisions(null);
+//            $report->setReasonForNoDecisions(null);
```

Comment:
Was this meant to be removed permanently, or just commented out for testing?

Reply:
We commented it out for testing, but I think it's pretty redundant now so I'll remove it permanently